package.json

```diff
@@ -23,29 +23,29 @@
   },
   "homepage": "https://github.com/fagbokforlaget/nestjs-audit-logging#readme",
   "devDependencies": {
-    "@nestjs/platform-express": "^8.4.4",
-    "@nestjs/testing": "^8.4.4",
-    "@types/jest": "^27.4.1",
+    "@nestjs/platform-express": "^10.2.6",
```

Comment:
@wrola  I'm wondering why we have these package in the first place - maybe we should think about removing it with some additional refactoring. Anyway, that's not something important for this PR, it's just a thought :)